<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerar ANOVA</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>box_ico.ico</normaloff>box_ico.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tabela_anova">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>451</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Digite o nome do arquivo ou o caminho:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nome_arquivo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_abrir">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Abrir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gerar_anova">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>420</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gerar ANOVA</string>
    </property>
   </widget>
  </widget>
  <action name="actionAbrir_Arquivo">
   <property name="text">
    <string>Abrir Arquivo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
